### PR TITLE
Feature/tao 6783/allow ux setup data folder

### PR DESCRIPTION
**Pull request by ekkinox**
- [x] https://github.com/oat-sa/generis/pull/571

### CODE REVIEW

**Comment by ekkinox** [comment]
Please merge https://github.com/oat-sa/generis/pull/571 first, due to new generic version dependency (8.2.0)

**Comment by rkarolis** [comment]
Works on Windows well.

File: install/services/class.CheckFileSystemComponentService.php [hunk 1]
```diff
@@ -1,121 +1,127 @@
...
+									   common_configuration_Component $component){
+
+		$content = json_decode($data->getContent(), true);
+        $rights = $content['value']['rights'];
+        $id = $content['value']['id'];
+        
+        $data = array('type' => 'FileSystemComponentReport',
+                      'value' => array('status' => $report->getStatusAsString(),
+                                       'message' => $report->getMessage(),
+        							   'id' => $id,
```
[the remark this answers is not in the document]
Comment (ekkinox) [reply install/services/class.CheckFileSystemComponentService.php]: done

File: install/js/step_server_setup.js [hunk 2]
```diff
@@ -62,9 +62,63 @@ require(['config'], function() {
...
+            // set a spinner up.
+            $serverSetup = $('#server-setup');
+            $serverSetup.css('visibility', 'visible').html('<span>' + $serverSetup.attr('data-next') + '</span>');
+            var spinner = new Spinner(getSpinnerOptions('small')).spin($serverSetup[0]);
+
+            setTimeout(function() { // Fake additional delay for user - 500ms.
+                var file_path = install.getData('file_path');
+                var file_path_overwrite = install.getData('file_path_overwrite');
+
+                console.log(file_path_overwrite);
```
[the remark this answers is not in the document]
Comment (ekkinox) [reply install/js/step_server_setup.js]: done

File: manifest.php [hunk 3]
```diff
@@ -45,10 +45,10 @@
     'label' => 'TAO Base',
     'description' => 'TAO meta-extension',
     'license' => 'GPL-2.0',
-    'version' => '22.14.5',
+    'version' => '22.14.6',
```
Comment (zagovorichev) [inline manifest.php]: this is feature, increase version, please

Comment (ekkinox) [reply manifest.php]: done, 24.1.0

**Pull request merged**